clio-integration-test/src/main/python/clio_test.py

```diff
@@ -64,68 +71,63 @@ def test_health(allowYellowHealth=False):
 
 
-def __assert_expected_fields(expected_fields, mapping_properties):
-    for (field_name, field_type) in expected_fields.items():
-        assert mapping_properties[field_name]['type'] == field_type
-    assert len(expected_fields) == len(mapping_properties)
-
-
 def test_read_group_mapping():
-    r = requests.get(elasticsearch_http_uri + '/read_group/_mapping/default')
-    js = r.json()
-    mappings = js['read_group']['mappings']['default']
-    assert mappings['dynamic'] == 'false'
-    expected_fields = {
-        'flowcell_barcode': 'keyword',
-        'lane': 'integer',
-        'library_name': 'keyword',
-        'location' : 'keyword',
-        'analysis_type': 'keyword',
-        'bait_intervals': 'keyword',
-        'data_type': 'keyword',
-        'individual_alias': 'keyword',
-        'initiative': 'keyword',
-        'lc_set': 'keyword',
-        'library_type': 'keyword',
-        'machine_name': 'keyword',
-        'molecular_barcode_name': 'keyword',
-        'molecular_barcode_sequence': 'keyword',
-        'paired_run': 'boolean',
-        'product_family': 'keyword',
-        'product_name': 'keyword',
-        'product_order_id': 'keyword',
-        'product_part_number': 'keyword',
-        'project': 'keyword',
-        'read_structure': 'keyword',
-        'research_project_id': 'keyword',
-        'research_project_name': 'keyword',
-        'root_sample_id': 'keyword',
-        'run_date': 'date',
-        'run_name': 'keyword',
-        'sample_alias': 'keyword',
-        'sample_gender': 'keyword',
-        'sample_id': 'keyword',
-        'sample_lsid': 'keyword',
-        'sample_type': 'keyword',
-        'target_intervals': 'keyword',
-        'ubam_md5': 'keyword',
-        'ubam_path': 'keyword',
-        'ubam_size': 'long',
-    }
-    __assert_expected_fields(expected_fields, mappings['properties'])
+    keywords = ['flowcell_barcode',
+                'library_name',
+                'location',
+                'analysis_type',
+                'bait_intervals',
+                'data_type',
+                'individual_alias',
+                'initiative',
+                'lc_set',
+                'library_type',
+                'machine_name',
+                'molecular_barcode_name',
+                'molecular_barcode_sequence',
+                'product_family',
+                'product_name',
+                'product_order_id',
+                'product_part_number',
+                'project',
+                'read_structure',
+                'research_project_id',
+                'research_project_name',
+                'root_sample_id',
+                'run_name',
+                'sample_alias',
+                'sample_gender',
+                'sample_id',
+                'sample_lsid',
+                'sample_type',
+                'target_intervals',
+                'ubam_md5',
+                'ubam_path']
+    expected = {k: 'keyword' for k in keywords}
```

Comment:
This is an improvement. If the tests were written in Scala than we'd have a hope of getting rid of this list of keywords from the test.